client/src/components/HostViews/Vote.vue

```diff
@@ -86,19 +68,27 @@ export default {
     })
     this.calculatePercentage()
 
-    this.emitCandidateList()
+    this.emitPlayerResponses()
     this.socketInstance.on('broadcast-game-state', () => {
-      this.emitCandidateList()
+      this.emitPlayerResponses()
     })
     this.socketInstance.on('ballot-collector', (playerBallot) => {
       if (this.pollsClosed) return
       this.countVotes(playerBallot)
     })
   },
   methods: {
-    emitCandidateList() {
-      this.socketInstance.emit('candidate-list', this.candidates
-        .map(candidate => candidate.player.name))
+    votePercentage(clientId) {
+      const CANDIDATE = this.candidates.find(candidate => candidate.player.clientId === clientId)
+      return CANDIDATE.votes
+    },
+    response(clientId) {
+      const playerResponse = this.promptResponses.find(response => response.player.clientId === clientId)
```

Comment:
Duplicate code? Can we re-use instead of copying and pasting?

Reply:
I dont see which code is being duplicated

Reply:
Got rid of the duplication. thanks for catching!